test/format/salvage.c

```diff
@@ -58,7 +58,8 @@ corrupt(void)
     wt_off_t offset;
     size_t len, nw;
     int fd, ret;
-    char buf[8 * 1024], copycmd[2 * 1024];
+    char copycmd[2 * 1024], path[1024];
```

Comment:
Is changing the size of `buf/path` intentional?

Reply:
I assume this is okay now as originally we were doing a large corruption, 8*1024, which is why buffer was large. But now as its only being used for the path we don't need it to be longer than 1024.